images/installer/Dockerfile.upi.ci

```diff
@@ -87,6 +90,11 @@ RUN mkdir /output && HOME=/output && \
     ibmcloud version && \
     ibmcloud plugin list
 
+# Install VMware plugin for powershell.  Create settings directory /output/.local/share/VMware/PowerCLI
+RUN pwsh -Command 'Install-Module VMware.PowerCLI -Force -Scope AllUsers' && \
+    pwsh -Command 'Install-Module -Name EPS -RequiredVersion 1.0 -Force -Scope AllUsers' && \
+    mkdir -p /output/.local/share/VMware/PowerCLI && chmod -R 777 /output/.local
```

Comment:
what's that mkdir for? I am assuming its for powercli but wouldn't that be before the `Import-Module`?